src/common/evaluation/QA/eval_depthmap_models/eval_main.py

```diff
@@ -49,6 +50,13 @@
                    input_location=os.path.join(MODEL_CONFIG.INPUT_LOCATION, MODEL_CONFIG.NAME),
                    output_location=temp_path)
 
+    # Copy filter to temp folder
+    download_model(ws=ws,
+                   experiment_name=FILTER_CONFIG.EXPERIMENT_NAME,
+                   run_id=FILTER_CONFIG.RUN_ID,
+                   input_location=os.path.join(FILTER_CONFIG.INPUT_LOCATION, MODEL_CONFIG.NAME),
+                   output_location=str(temp_path) + '/' + FILTER_CONFIG.NAME)
```

Comment:
The better usage of pathlib would be:

```suggestion
                   output_location=str(temp_path / FILTER_CONFIG.NAME))
```